packages/flutter/lib/src/rendering/object.dart

```diff
@@ -624,7 +624,7 @@ class _RootSemanticsFragment extends _InterestingSemanticsFragment {
     assert(parentSemantics == null);
     owner._semantics ??= new SemanticsNode.root(
       handler: owner is SemanticActionHandler ? owner as dynamic : null,
-      owner: owner.owner
+      owner: owner.owner.semanticsOwner
```

Comment:
We should rename the `_SemanticsFragment.owner` field to `renderObjectOwner` and rename the RenderObject.owner field to `pipelineOwner`. Though I guess the latter will affect AbstractNode? hm.

Anyway that can all wait for some other patch. It's just weird seeing owner.owner.semanticsOwner.